src/main/java/core/basesyntax/OperationStrategyImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Map;

```

Comment:
It's recommended to use the Strategy pattern without switch-cases or if-else constructions. Ensure that the `getHandler` method handles the case when there is no handler for the provided operation, possibly by throwing an exception with an informative message.